### PR TITLE
Fix two issues with new 2020 ENTSO-E grid extract

### PR DESCRIPTION
**Pull request by fneum**
## Changes proposed in this Pull Request

There were two issues with the 2020 ENTSO-E grid extract from https://www.entsoe.eu/data/map/ preventing clustering down to 37 nodes (39 was the minimum because additional subnetworks not connected to AC network or cross-country HVDC links were formed).

**Skagerrak HVDC links between DK and NO** were partially allocated to the wrong terminal bus in NO and included a "self-link".

For **HVDC connections mainland IT, Sardinia and Corsica** a node was wrongly created at the intersection between two actually separate connections.

Note 22 Aug 2020

## Checklist

- [x] I tested my contribution locally and it seems to work fine.
~- [ ] Code and workflow changes are sufficiently documented.~
~- [ ] Newly introduced dependencies are added to `environment.yaml` and `environment.docs.yaml`.~
~- [ ] Changes in configuration options are added in all of `config.default.yaml`, `config.tutorial.yaml`, and `test/config.test1.yaml`.~
~- [ ] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.~
- [x] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes.

### CODE REVIEW

File: scripts/base_network.py [hunk 1]
```diff
@@ -548,6 +555,8 @@ def base_network():
 
     _apply_parameter_corrections(n)
 
+    _apply_sardinia_corsica_link_corrections(n)
+
```
Comment (FabianHofmann) [inline scripts/base_network.py:559]:
```suggestion
```
Now, `n = _remove_unconnected_components(n)` has to be placed below `_apply_parameter_corrections(n)` and bus '7276' is removed

**Review by FabianHofmann — COMMENTED** [review]
Great that you figured out the problems! I don't love the hard coded bit though. I made a suggestion how we can avoid the hard-coded ` _apply_sardinia_corsica_link_corrections(n)` function.

**Comment by fneum** [comment]
Ah, this trick is very neat! I also thought the hardcoded bit was not the nicest, but couldn't think of this alternative.

**Review by FabianHofmann — APPROVED** [review]
Nice! I think as soon you double-checked if the assignment is correct, let's merge it.

**Comment by fneum** [comment]
Behaviour is the same as in the previous hardcoded version.

**Pull request merged**